hack/build-manifests.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -e
+
+namespace=${NAMESPACE:-kube-system}
+
+
+bridge_marker_image_repo=${BRIDGE_MARKER_IMAGE_REPO:-quay.io/kubevirt}
+bridge_marker_image_name=${BRIDGE_MARKER_IMAGE_NAME:-bridge-marker}
+bridge_marker_image_version=${BRIDGE_MARKER_IMAGE_VERSION:-latest}
+bridge_marker_image_pull_policy=${BRIDGE_MARKER_IMAGE_PULL_POLICY:-Always}
+
+
+openshift_node_image_repo=${OPENSHIFT_NODE_IMAGE_REPO:-docker.io/openshift}
```

Comment:
This was used for ovs-vsctl dependency, I don't see it being used in this project.

Reply:
Done